runhouse/resources/hardware/cluster_factory.py

```diff
@@ -2,17 +2,29 @@
 from typing import Dict, List, Optional, Union
 
 from runhouse.resources.hardware.utils import RESERVED_SYSTEM_NAMES
+from runhouse.rns.utils.api import relative_ssh_path
 
-from .cluster import Cluster
+from .cluster import Cluster, ServerConnectionType
 from .on_demand_cluster import OnDemandCluster
 from .sagemaker_cluster import SageMakerCluster
 
+SSH_CONN = ServerConnectionType.SSH.value
+HTTPS_CONN = ServerConnectionType.TLS.value
+HTTP_CONN = ServerConnectionType.NONE.value
+PARAMIKO_CONN = ServerConnectionType.PARAMIKO.value
+AWS_SSM_CONN = ServerConnectionType.AWS_SSM.value
```

Comment:
We don't need these, you can use the enum directly and it will act like a string when needed.